src/NzbDrone.Core/Notifications/Stash/StashSettings.cs

```diff
@@ -65,6 +65,9 @@ public StashSettings()
         [FieldDefinition(10, Label = "Map Paths To", Type = FieldType.Textbox, Advanced = true, HelpText = "Stash Path, Used to modify site paths when Stash sees library path location differently from Whisparr")]
         public string MapTo { get; set; }
 
+        [FieldDefinition(11, Label = "Run Identify Task", Type = FieldType.Textbox, Advanced = true, HelpText = "Run metadataIdentify task with StashDB/Autotag on new files")]
```

Comment:
Shouldn't this be a `bool` and `FieldType.Checkbox`

Reply:
Yep, thanks for catching that.